pywinauto/unittests/test_application.py

```diff
@@ -46,6 +46,9 @@
 import ctypes
 import mock
 
+from pywinauto.controls.common_controls import TrackbarWrapper
+from pywinauto.controls.uiawrapper import UIAWrapper

```

Comment:
Pls remember that 'UIA' is still optional we use it directly only in the dedicated files like test_uiawrapper.py  Try to see if your unit tests can be split into win32 and uia parts. Use `if UIA_support:` to verify. See the failing unit test on AppVeyor:
```
Traceback (most recent call last):
  File "c:\python27\lib\unittest\case.py", line 329, in run
    testMethod()
  File "c:\python27\lib\site-packages\nose\loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "c:\python27\lib\site-packages\nose\importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "c:\python27\lib\site-packages\nose\importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "C:\projects\pywinauto\pywinauto\unittests\test_application.py", line 50, in <module>
    from pywinauto.controls.uiawrapper import UIAWrapper
  File "C:\projects\pywinauto\pywinauto\controls\uiawrapper.py", line 40, in <module>
    import comtypes
ImportError: No module named comtypes
```

Reply:
Thanks I'll have a look at it, Pycharm autoimports got me ^^